relnotes/smartunionformatter.feature.md

```diff
@@ -0,0 +1,8 @@
+### New `SmartUnion` formatting function
+
+`ocean.text.convert.SmartUnion`
+
+The helper function `formatActive` accepts a smart-union and returns a struct
```

Comment:
Maybe `asActive` to match https://github.com/sociomantic-tsunami/ocean/blob/v3.x.x/src/ocean/text/convert/DateTime_tango.d#L968 ?

Reply:
Hmm... that's not a name that makes any sense to the casual reader.

Reply:
Well, having `format` as part of name is definitely redundant and confusing as you can only ever use this wrapper with `(s)format` calls. It affects representation, not actually formats stuff.

I am not sure what would be the best naming pattern though.

Reply:
I'd suggest simply `activeField`, so the usage would look like: `sformat(buf, "union active member: {}", activeField(su));`

Reply:
How would you call the time one? I'd prefer to have consistent naming style for all formatter wrappers if possible.

Reply:
Maybe:
```D
sformat(buf, "the time is: {}", prettyString(time));
```

Reply:
Hm, how is that any clearer than `asActiveField` and `asPrettyString` though? From your naming one may expect that field/string gets returned eagerly and passed to formatter after that. At least that is how I read it.

Reply:
I don't really feel the `as` part adds much meaning, but I'm fine with calling it `asActiveField`, for the sake of consistency.

Reply:
I find it useful to clearly denote "eager vs lazy" in either module names or function names, similar to how there is `split` and `splitter` in Phobos (ignoring stupid name choice itself :P)